src/controllers/profile.controller.ts

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable object-curly-newline */
+/* eslint-disable quotes */
+/* eslint-disable import/prefer-default-export */
+import { Request, Response } from 'express';
+import { UserService } from '../services/user.services';
+import uploadFile from '../helper/imageUploader';
+
+
+
+export const getProfileController = async (req: Request, res: Response) => {
+  try {
+    const userId = (req as any).user.id;
+    const profile = await UserService.getProfileServices(userId);
```

Comment:
getProfileService works the same with getYseByid you can check and reuse it instead of writing new one